suite/tests/CMakeLists.txt

```diff
@@ -3742,6 +3742,46 @@ foreach (run ${run_list})
   endif (enabled)
 endforeach(run)
 
+if (NOT DEBUG AND UNIX AND X64 AND NOT CMAKE_COMPILER_IS_CLANG)
+  set_tests_properties(
+    code_api|tool.drcacheoff.simple
+    code_api|tool.drcachesim.simple
+    code_api|tool.drcachesim.simple-config-file
+    code_api|tool.drcachesim.TLB-simple
+    code_api|tool.drcachesim.missfile
+    code_api|tool.drcachesim.missfile-config-file
+    code_api|tool.drcachesim.phys
+    code_api|tool.drcachesim.filter-simple
+    code_api|tool.drcachesim.filter-no-i
+    code_api|tool.drcachesim.filter-no-d
+    code_api|tool.drcachesim.delay-simple
+    code_api|tool.drcachesim.warmup-valid
+    code_api|tool.drcachesim.warmup-zeros
+    code_api|tool.drcachesim.threads
+    code_api|tool.drcachesim.threads-with-config-file
+    code_api|tool.drcachesim.TLB-threads
+    code_api|tool.drcachesim.multiproc
+    code_api|tool.drcachesim.miss_analyzer
+    code_api|tool.drcacheoff.simple
```

Comment:
There seem to be duplicate lines here: this is the same as 3747.
Please verify the rest are all unique.

Reply:
Done.